ios/AddCalendarEvent.m

```diff
@@ -272,6 +273,33 @@ - (EKEvent *)createNewEventInstance {
     if (options[_allDay]) {
         event.allDay = [RCTConvert BOOL:options[_allDay]];
     }
+    if (options[_alert]) {
+        NSDate *originalDate =  [RCTConvert NSDate:options[_startDate]];
+
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"0"] == NSOrderedSame) 
+        { 
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:originalDate];
```

Comment:
you're using `alarmWithAbsoluteDate` - let's consider the following scenario: user

1) creates an event for 2pm and adds an alert that should fire 30 minutes before
2) moves the event to 1:30pm
3) when does the alert fire and is it what the user expects?

I think [this](https://developer.apple.com/documentation/eventkit/ekalarm/1507338-alarmwithrelativeoffset?language=objc) might be useful.